docs/changelog.md

```diff
@@ -134,6 +134,22 @@ Support for Gryo MessageSerializer [has been dropped in TinkerPop 3.6.0](https:/
 and we therefore also no longer support it in JanusGraph.
 GraphBinary is now used as the default MessageSerializer.hb
 
+##### Batch Processing enabled by default
```

Comment:
Shall we call it `limited batch processing enabled by default`, since we now also enable limited batch processing?

Reply:
I don't have a strong opinion on this, but I'd opted to just mention that batch processing itself is enabled by default as that is the major change in my opinion. I think the _limited_ part is only a configuration to let the batch processing behave better.

Reply:
Sounds fair!